### DISCUSSION TITLE
Does it have a `<Head>` component?

### DISCUSSION THREAD

**Discussion by ojvribeiro** [unjs/unhead#149] [Q&A] [answered]
Hi UnJS guys, nice work you're doing with this ecosystem.

My question is:

Does Unhead have a `<Head>` component that can set head tags in the template? Since `@vueuse/head` is being deprecated and Unhead is being encouraged, this is a nice feature to have. But I couldn't find it in docs unfortunately.

**Comment by ojvribeiro** [comment]
Oh, found it 😭 

![image](https://user-images.githubusercontent.com/8026741/234013852-49aa2700-df3b-41a8-a1db-3ac53adfd726.png)

You guys plan to include it as @vueuse/head is being deprecated?

**Reply by harlan-zw** [reply, marked as answer]
Yes it's planned 🙂

**Reply by ojvribeiro** [reply]
Much love ♥

**Comment by harlan-zw** [comment]
FYI it now ships a `<Head>` component, see https://unhead.unjs.io/setup/vue/components

**Reply by ojvribeiro** [reply]
Yeah, I'm already using `unhead` thanks to this. 😄 (see  https://github.com/ojvribeiro/vulmix/pull/229)

Much love for all your work on this ♥
